test-requirements.txt

```diff
@@ -17,7 +17,7 @@ pytest-cov==2.6.0
 rstcheck==3.2
 scikit-learn==0.20.2
 scipy==1.2.1
-tensorflow==1.12.0
+tensorflow==1.13.1
```

Comment:
Which tests need this upgrade?

Reply:
I couldn't run `pip install -r test-requirements.txt` without this change.

Reply:
^I think this issue (needing to bump the TF version) would have been addressed by this PR btw, which updates our contributor guide to standardize dependencies/Python versions/tests run locally & on Travis: https://github.com/mlflow/mlflow/pull/1097. Specifically I think the issue is that TF 1.12.0 isn't available in certain Python versions (like 3.7)

Reply:
Good to bump the dependency version anyways to make sure we're testing against the latest TF